client/client.go

```diff
@@ -0,0 +1,182 @@
+package client
```

Comment:
should prolly move these to a new repo. From a cursory look it is needed for the cli tool?

Reply:
yes. ok.

Reply:
done